src/main/java/com/ajou/hertz/domain/user/service/UserProfileImageCommandService.java

```diff
@@ -20,38 +19,55 @@
 @Service
 public class UserProfileImageCommandService {
 
-	private final UserQueryService userQueryService;
 	private final FileService fileService;
 	private final UserProfileImageRepository userProfileImageRepository;
 
+	private static final String USER_PROFILE_IMAGE_UPLOAD_PATH = "user-profile-images/";
+
 	/**
 	 * 유저의 프로필 이미지를 업데이트한다.
 	 *
-	 * @param userId 유저 id
+	 * @param user 프로필 이미지를 업데이트할 유저
 	 * @param newProfileImage 새로운 프로필 이미지
 	 *
-	 * @return 업데이트된 유저 정보
+	 * @return 새로운 프로필 이미지 URL
 	 */
-	public UserDto updateProfileImage(Long userId, MultipartFile newProfileImage) {
-
-		User user = userQueryService.getById(userId);
-		Optional<UserProfileImage> optionalOldProfileImage = userProfileImageRepository.findById(userId);
+	public String updateProfileImage(User user, MultipartFile newProfileImage) {
+		deleteOldProfileImage(user.getId());
+		UserProfileImage newUserProfileImage = uploadNewProfileImage(user, newProfileImage);
```

Comment:
혹시 flush를 이 부분으로 옮겨주실 수 있을까요?
이유는 이따가 따로 말씀드릴게요

```suggestion
		deleteOldProfileImage(user.getId());
		userProfileImageRepository.flush();
		UserProfileImage newUserProfileImage = uploadNewProfileImage(user, newProfileImage);
```